src/System.Private.ServiceModel/tools/test/SelfHostWcfService/TestResources/WebSocketResources.cs

```diff
@@ -200,15 +200,243 @@ protected override int GetPort(ResourceRequestContext context)
 
         protected override Binding GetBinding()
         {
-            int DefaultMaxReceivedMessageSize = 64 * 1024 * 1024;
-
-            NetHttpBinding binding = new NetHttpBinding();
+            NetHttpBinding binding = new NetHttpBinding()
+            {
+                MaxReceivedMessageSize = DefaultMaxReceivedMessageSize,
+                MaxBufferSize = DefaultMaxReceivedMessageSize,
+            };
             binding.WebSocketSettings.TransportUsage = WebSocketTransportUsage.Always;
-            binding.MaxReceivedMessageSize = DefaultMaxReceivedMessageSize;
-            binding.MaxBufferSize = DefaultMaxReceivedMessageSize;
             binding.TransferMode = TransferMode.Streamed;
```

Comment:
For the future: would prefer the changes above (cleaning up the NetHttpBinding declarations) be in a separate commit even if it's in the same PR, but don't spend time splitting the changes now.